text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/tag/resolver/Formatter.java

```diff
@@ -139,4 +143,53 @@ public static TagResolver booleanChoice(@TagPattern final @NotNull String key, f
       return Tag.inserting(context.deserialize(value ? trueCase : falseCase));
     });
   }
+
+  /**
+   * Creates a replacement that inserts a list of components. These components are joined together by {@link Component#join(JoinConfiguration.Builder, ComponentLike...)}.
+   *
+   * <p>This tag expects a separator as the first argument. A last separator may optionally be provided.</p>
```

Comment:
Can this be optional too? In the case where you're happy to just use `JoinConfiguration.noSeparators`?